packages/next-dashboard/src/utils/createDashboardHOC.js

```diff
@@ -98,16 +100,21 @@ export default function createDashboardHOC({
   needAuthDefault,
   unauthedRoute,
   error: errorConf,
-  themes: confThemes,
+  themes: themesConf,
+  branding: brandingConf,
 }: Config): <U: {}, Q: {} = {}>(
   Comp: DashboardComponent<U, Q>,
   needAuth?: boolean,
 ) => NextComponent<WrappedUnified<U, Q>, InitialUnified<Q>> {
-  const themes: Theme[] = confThemes || [
+  const themes: Theme[] = themesConf || [
```

Comment:
You're calling my naming scheme inconsistent? >.>

Reply:
Yes :p

so I decided to fix it up whilst I'm touching this code anyway